src/Microsoft.AspNetCore.Http.Abstractions/Authentication/AuthenticationManager.cs

```diff
@@ -20,18 +20,11 @@ public abstract class AuthenticationManager
 
         public abstract IEnumerable<AuthenticationDescription> GetAuthenticationSchemes();
 
-        public abstract Task AuthenticateAsync(AuthenticateContext context);
+        public abstract Task<AuthenticateInfo> GetAuthenticateInfoAsync(string authenticationScheme);
```

Comment:
Switching the names would be a major break that we want to avoid. Not nearly as many people will use this version.